third-party/realdds/src/dds-stream.cpp

```diff
@@ -31,10 +36,44 @@ void dds_stream::open( std::string const & topic_name, std::shared_ptr< dds_subs
     //Topics with same name and type can be created multiple times (multiple open() calls) without an error.
     auto topic = topics::device::image::create_topic( subscriber->get_participant(), topic_name.c_str() );
 
-    //To support automatic streaming (without the need to handle start/stop-streaming commands) the reader is created
+    //To support automatic streaming (without the need to handle start/stop-streaming commands) the readers are created
```

Comment:
Now we have automatic streaming?

Reply:
No, but Eran is working on it. The comment was there from before.
Changed it to plural when I used two readers in one stream object, but after our discussion I returned to one reader, will restore the comment the way it was before